src/main/java/frame/panels/TransactionsPanel.java

```diff
@@ -182,7 +193,8 @@ private boolean hasEnoughMoney()
     private boolean fieldsAreValid()
     {
         boolean validTransactionDate = transactionDateInput.getText() != null;
-        // todo: also confirm that the date is today or earlier
+        // todo: also confirm that the date is today
```

Comment:
I think this comment can be removed, there shouldn't be a way for this problem to come up

Reply:
also gonna get rid of the validTransactionDate check - thanks for catching that